pymc3/backends/base.py

```diff
@@ -463,6 +465,10 @@ def point(self, idx, chain=None):
             chain = self.chains[-1]
         return self._straces[chain].point(idx)
 
+    @property
+    def points(self):
```

Comment:
Are there situations where we'd want to limit this to only some chains or so? If so, it might be better to make this a function, that way we can add keyword args later.
Also, missing docs ;-)